website/docs/r/service_account.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_service_account"
+sidebar_current: "docs-vcd-resource-service_account"
+description: |-
+  Provides a resource to manage Service Accounts. Service Accounts can have defined roles
+  and act just like a VCD user. Service Accounts, when activated, provide one-time use
+  access tokens for authentication to the VCD API, during which a new access token is generated.
+---
+
+# vcd\_service\_account 
+
+Supported in provider *v3.10+* and VCD 10.4+.
+
+Provides a resource to manage Service Accounts. Service Accounts can have defined roles
+and act just like a VCD user. Service Accounts, when activated, provide one-time use
+access tokens for authentication to the VCD API, during which a new access token is generated.
+Explained in more detail [here].[service-accounts]
```

Comment:
Preview tool doesn't like this.

https://registry.terraform.io/tools/doc-preview

<img width="722" alt="image" src="https://github.com/vmware/terraform-provider-vcd/assets/15804230/9e43a102-cfae-46ea-a454-f45306f99ed3">

Reply:
Thanks for spotting, fixed.